Jenkinsfile

```diff
@@ -49,6 +49,18 @@ pipeline {
         }
       }
     }
+
+    stage('RHEL8 Integration tests') {
+      steps {
+        sh './bin/test_rhel8_integration'
```

Comment:
in terms of naming it looks cleaner imo to have the platform at the end so it becomes `test_rhel8_integration` -> `test_integration_rhel8`. That way, at a first glance it is clear this script is for rhel8